NEWS.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
```suggestion
# R Development Container 0.1

First version of the R Development Container, including:

- DevContainer definition for creating a codespace allowing users to build R in a web based environment
- Instructions for building the current version of R from the SVN trunk within the DevContainer in the README.md file
- Contributor workflow instructions in the README.md file
- Docker image build from the dev-container branch which contains the requirements for building base R. The image is available from https://github.com/r-devel/r-dev-env/pkgs/container/r-dev-env
- Project website build from the website branch and available at https://contributor.r-project.org/r-dev-env/

We hope the R Development Container helps you in your R journey 😊
```

The release information can go here, following the use of NEWS.md for other packages (e.g., https://github.com/tidyverse/ggplot2/blob/main/NEWS.md).